docs/imagecustomizer/concepts/iso.md

```diff
@@ -8,23 +8,52 @@ nav_order: 2
 
```

Comment:
You are making quite substantial edits to this file. Can we split this off into a separate change?